providers/pinpoint/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [0.2.5](https://github.com/novuhq/novu/compare/v0.2.4...v0.2.5) (2021-11-05)
+
+**Note:** Version bump only for package @notifire/ses (now @novu/ses)
```

Comment:
This information doesn't seem accurate to what you are adding.

Reply:
Do I just leave that file as it is now, since I'm not the one creating a new release?